src/Integrations.Umbraco/Integrations.Umbraco.csproj

```diff
@@ -35,12 +35,11 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<!--Version 15-16-->
-		<PackageReference Include="Umbraco.Cms.Web.Common" Version="[15.0.0,17)" />
-		<PackageReference Include="Umbraco.Cms.Web.Website" Version="[15.0.0,17)" />
-		<PackageReference Include="Umbraco.Cms.Api.Common" Version="[15.0.0,17)" />
-		<PackageReference Include="Umbraco.Cms.Api.Management" Version="[15.0.0,17)" />
-
+		<!--Version 17-->
```

Comment:
Because of the change in `Microsoft.OpenApi` I had to bump this to min version 17 to get it to work